scripts/emr/terraform/emr-spark.tf

```diff
@@ -0,0 +1,69 @@
+# Marks AWS as a resource provider.
+provider "aws" {
+  access_key = "${var.access_key}"
+  secret_key = "${var.secret_key}"
+  region     = "${var.region}"
+}
+
+# `aws_emr_cluster` is built-in to Terraform. We name ours `emrSparkCluster`.
+resource "aws_emr_cluster" "emrSparkCluster" {
```

Comment:
Minor style point: Camel case seems to be less common than separating words with hyphens and use all lower case (e.g. `"emr-spark-cluster"`).

Reply:
That's the case, yeah.